scan-manager/docs/setup_artifacts/deployment/config.yml

```diff
@@ -3,31 +3,31 @@ scikit_0.23:
   deployment: scikit_0.23_deployment.yml
   default_base_image:
     name: python36_scikit
-    value: gcr.io/certifai-dev/certifai-model-container:1
+    value: c12e/cortex-certifai-model-scikit:v3
   available_base_images:
   - name: python36_scikit
-    value: gcr.io/certifai-dev/certifai-model-container:1
+    value: c12e/cortex-certifai-model-scikit:v3
 h2o_mojo:
   deployment: h2o_mojo_deployment.yml
   default_base_image:
     name: h2o_dai
-    value: gcr.io/certifai-dev/certifai-h2o-model-container:1
+    value: c12e/cortex-certifai-model-h2o-mojo:v3
   available_base_images:
   - name: h2o_dai
-    value: gcr.io/certifai-dev/certifai-h2o-model-container:1
+    value: c12e/cortex-certifai-model-h2o-mojo:v3
 r_model:
   deployment: r_model_deployment.yml
   default_base_image:
     name: r_model
-    value: gcr.io/certifai-dev/certifai-r-model-container:2
+    value: c12e/cortex-certifai-model-r:v2
   available_base_images:
   - name: r_model
-    value: gcr.io/certifai-dev/certifai-r-model-container:2
-proxied_model:
+    value: c12e/cortex-certifai-model-r:v2
+hosted_model:
   deployment: hosted_model_proxy_deployment.yml
   default_base_image:
     name: certifai-proxy
-    value: gcr.io/certifai-dev/certifai-proxy-default:1
+    value: c12e/cortex-certifai-hosted-model:v1
```

Comment:
@akumar-cs Can we have the repo created with the same name as above?

Reply:
Repo created